### DISCUSSION TITLE
SDXL - Comfy UI missing CLIPTextEncodeSDXL and CLIPTextEncodeSDXLRefiner

### DISCUSSION THREAD

**Discussion by vsdrake20** [AbdBarho/stable-diffusion-webui-docker#562] [Q&A]
Would anyone happen to have some insight on this issue? They are nodes and I have seen them used a bunch but they are missing for me. People seem to be able to use the "out of the box" but man I am lost cause they are missing for me and there seems to be no information online. Any help would be much appreciated.

Example: https://youtu.be/2Xe79Nl_6jA?t=99

**Comment by AbdBarho** [comment]
Comfy UI version in this repo is a bit outdated.
